README.md

```diff
@@ -1,47 +1,99 @@
-Sugar Toolkit
-=============
+# Sugar Toolkit GTK3
 
-Sugar Toolkit provides services and a set of GTK+ widgets to build
-activities and other Sugar components on Linux based computers.
+Sugar Toolkit provides services and a set of GTK+ widgets to build activities and other Sugar components on Linux-based computers. This toolkit is designed to work seamlessly with the Sugar Learning Environment, enabling developers to create educational applications with intuitive, child-friendly interfaces.
 
-This is the GTK+ 3 binding of the Sugar Toolkit.
+---
 
-https://www.sugarlabs.org/
+## Features
+- **Pre-built GTK+ Widgets**: Simplify UI design for activities.
+- **Integration with Sugar Environment**: Access system services, datastore, and collaboration tools.
+- **Lightweight and Modular**: Build standalone activities or integrate into the Sugar desktop.
 
-https://wiki.sugarlabs.org/
+For more information, visit:
+- [Sugar Labs Official Website](https://www.sugarlabs.org/)
+- [Sugar Labs Wiki](https://wiki.sugarlabs.org/)
 
-Installing on Debian or Ubuntu
-------------------------------
+---
 
-Automatically done when you install [Sugar
-desktop](https://github.com/sugarlabs/sugar).
+## Installation
 
-To install Sugar Toolkit alone without Sugar desktop,
+### Debian/Ubuntu
+Sugar Toolkit GTK3 is automatically installed with [Sugar desktop](https://github.com/sugarlabs/sugar).
+To install Sugar Toolkit alone without Sugar desktop:
 
-```
+```bash
 sudo apt install python3-sugar3
 ```
 
-Installing on Fedora
---------------------
-
-Automatically done when you install [Sugar
-desktop](https://github.com/sugarlabs/sugar).
+### Fedora
+Similar to Debian/Ubuntu, it is included with the [Sugar desktop](https://github.com/sugarlabs/sugar).
+To install Sugar Toolkit alone without Sugar desktop:
 
-To install Sugar Toolkit alone without Sugar desktop,
-
-```
+```bash
 sudo dnf install sugar-toolkit-gtk3
 ```
 
-Building
---------
-
-Sugar Toolkit follows the [GNU Coding
-Standards](https://www.gnu.org/prep/standards/).
-
-Install all dependencies, especially sugar-artwork and
-sugar-datastore.
+## Building from Source
+Sugar Toolkit GTK3 follows the [GNU Coding Standards](https://www.gnu.org/prep/standards/).
+
+### Note
+Ensure the following dependencies are installed:
+- `sugar-artwork`
+- `sugar-datastore`
+- `GTK+ 3`
+
+### Steps
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/sugarlabs/sugar-toolkit-gtk3.git
+   cd sugar-toolkit-gtk3
+   ```
+2. Run the configuration script:
+   ```bash
+   ./autogen.sh
+   ```
+3. Build and install:
+   ```bash
+   sudo make clean
+   sudo make
+   sudo make install
+   ```
+   
+Alternatively, you can skip the `make clean` step with:
+```bash
+sudo ./autogen.sh
+sudo make
+sudo make install
+```
```

Comment:
configure accepts a `--with-python2` and `--with-python3` arguments which should be included.